fundraising/banner.config.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Banner configuration
+ *
+ * @licence GNU GPL v2+
+ * @author Kai Nissen <kai.nissen@wikimedia.de>
+ */
+( function( Banner ) {
+	'use strict';
+	
+	Banner.config = {
+		tracking: {
+			campaign: 'default',
+			keyword: 'default',
+			accessCode: 'default',
+			baseUrl: 'https://spenden.wikimedia.de/',
+			libUrl: 'https://spenden.wikimedia.de/piwik/piwik.js',
+			trackerUrl: 'https://spenden.wikimedia.de/piwik/piwik.php',
```

Comment:
this is not used right now. Would this be actually need to call PHP tracker API from JavaScript? I am not saying this should be changed/removed, I am just being (too) curious.